api/__init__.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask
+from config.config import app_config
```

Comment:
Consider refactoring your imports this way:

```
1. System imports.

2. Imports from packages installed.

3. Imports for other files on your projects file system.
```

e.g.

```
import sys # where applicable

from flask import Flask
from flask_sqlalchemy import SQLAlchemy

from config.config import app_config
```

NOTICE the space/ line between the imports. Its a recommended practise.